### DISCUSSION TITLE
Welcome to nPinA-pi-network Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/nPinA-pi-network#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TriCellsTechnologies** [comment]
I have an idea can we work on it?

**Reply by TriCellsTechnologies** [reply]
Like family tree on Pi Network

**Comment by TheCaveMembership** [comment]
About
Leading Web3 engagement with gamified NFT memberships as CEO of The Cave Collectables. Also developing Izza Pay: Seamlessly accept Pi Coins on Shopify & e-commerce platforms. Join us in revolutionizing payments and community building. #PiNetwork #PiOS

Info
Business Owner at The Cave Collectables
Toronto, Canada

Contact
[thecavecollectables.com/](https://thecavecollectables.com/)
[cam@thecavecollectables.com](mailto:cam@thecavecollectables.com)
[cameron-macdonald-7166b79a](https://www.linkedin.com/in/cameron-macdonald-7166b79a)
[@_CamMac](https://twitter.com/@_CamMac)
[961095344513491034](https://discord.com/users/961095344513491034)

**Comment by KOSASIH** [comment]
OK, I will try to build a collaborative project with you and all src code
will be saved on my git repo.
Before that, I want to know about the benefits for me, especially the
payment and also the sustainability of this project in the future.

Thank you, may God bless you and your family.



KOSASIH

Pada tanggal Kam, 23 Mei 2024 21.08, TheCaveMembership <
***@***.***> menulis:
